geoips/commandline/ancillary_info/alias_mapping.yaml

```diff
@@ -10,6 +10,10 @@ coverage-checkers:
   - cchk
   - cchks
   - coverage-checker
+databases:
```

Comment:
Are there any new docs accompanying the database interface?

Reply:
No...

Reply:
There are currently no actual examples of this - we just moved the interface over.  When we add simple database plugin examples, we will document at that time.  We did add database items to the CLI.